metadata-ingestion/setup.py

```diff
@@ -626,6 +627,7 @@
         "gcs = datahub.ingestion.source.gcs.gcs_source:GCSSource",
         "sql-queries = datahub.ingestion.source.sql_queries:SqlQueriesSource",
         "fivetran = datahub.ingestion.source.fivetran.fivetran:FivetranSource",
+        "qlik-cloud = datahub.ingestion.source.qlik_cloud.qlik_cloud:QlikCloudSource",
```

Comment:
does it make sense to call this `qlik`, or is there a reason you called it `qlik-cloud`

Reply:
Lets make it qlik-sense as they also have product qlik-view.